Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG version=12.6.0-alpine
+FROM node:$version
+
+ARG INCLUDE_WORKER_TEMPLATE=https://github.com/cloudflare/worker-template
```

Comment:
This is nice, but I wonder if this should be included in the official image. What's the use case for running the default template in a container?

Reply:
@Electroid Originally, I left it out, however after actually using the built image, it became impractical to use as the build/preview/publish commands take a long time to run (since it will have to download / install the components every execution.  Including the template here, "caches" those files so the commands are fast (I also thought it was inline with the project in that it assumes this template unless you specify another one as part of the config).  Just a note, I'm a novice when it comes to Node 
 so there is probably a much better / more elegant way to handle this - def open to suggestions.